wagtail/admin/templates/wagtailadmin/edit_handlers/help_panel.html

```diff
@@ -0,0 +1,6 @@
+<fieldset>
+	{% if self.heading %}
```

Comment:
This line is 2 spaces too indented.

Reply:
No, the rest was correctly indented, it’s only this line that had 2 extra spaces ;)

Reply:
Ah, I see the problem, you’re using tabs. Please use an indentation base of 4 spaces instead.